gnovm/tests/files/map40.gno

```diff
@@ -0,0 +1,9 @@
+package main
+
+func main() {
+	const s = ""
+	t := map[any]int{len(s) + 0: 0, 0: 1}
+}
+
+// TypeCheckError:
+// main/map40.gno:5:34: duplicate key 0 in map literal; main/map40.gno:5:2: declared and not used: t
```

Comment:
also: https://go.dev/play/p/BAolfdnpPxz?v=goprev

Reply:
we had same error I think